Docker setup is not working

### What happened?

Docker first-time setup is not working as expected and an error is thrown after running
```
docker-compose up --build
``` 



### Please reproduce in steps

Just follow the steps detailed in the [first time guide](https://github.com/kodadot/nft-gallery/blob/97469598cb8a261d5b05d476c8ea1193b643aeda/FIRST_TIME.md#docker-)

1.  Pull _main_ 
2. Create _.env_ with the following values
```
NUXT_ENV_KEYRING=true
PINATA_API_KEY=actual_value
PINATA_SECRET_API_KEY=actual_value
PINATA_MASTER=actual_value
```

3. Run `docker-compose up --build` 


### Expected Behavior

The server should run and a be available at [localhost:9090](http://localhost:9090/)

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

![telegram-cloud-photo-size-4-5866154109579673092-y](https://github.com/kodadot/nft-gallery/assets/44554284/12c16891-b488-4ad5-8c14-9f1884da0671)


### Relevant log output

```shell
[+] Running 1/0
 ✔ Container Kodadot  Created                                                                                                                                                                                                        0.0s 
Attaching to Kodadot
Kodadot  | 
Kodadot  | > kodadot-nft-marketplace@3.0.0 dev /app
Kodadot  | > nuxi prepare && cross-env NODE_OPTIONS=--openssl-legacy-provider PORT=9090 nuxi dev
Kodadot  | 
Kodadot  | Nuxi 3.0.0
Kodadot  | 
Kodadot  |  ERROR  Cannot find module '/app/node_modules/@kodadot1/static/dist/index.cjs'
Kodadot  | 
Kodadot  |   at createEsmNotFoundErr (node:internal/modules/cjs/loader:1098:15)
Kodadot  |   at finalizeEsmResolution (node:internal/modules/cjs/loader:1091:15)
Kodadot  |   at resolveExports (node:internal/modules/cjs/loader:567:14)
Kodadot  |   at Module._findPath (node:internal/modules/cjs/loader:636:31)
Kodadot  |   at Module._resolveFilename (node:internal/modules/cjs/loader:1063:27)
Kodadot  |   at Function.resolve (node:internal/modules/cjs/helpers:116:19)
Kodadot  |   at _resolve (node_modules/.pnpm/jiti@1.18.2/node_modules/jiti/dist/jiti.js:1:241025)
Kodadot  |   at jiti (node_modules/.pnpm/jiti@1.18.2/node_modules/jiti/dist/jiti.js:1:243309)
Kodadot  |   at utils/config/defineApolloConfig.ts:1:303
Kodadot  |   at jiti (node_modules/.pnpm/jiti@1.18.2/node_modules/jiti/dist/jiti.js:1:245784)
Kodadot  | 
Kodadot  |  ELIFECYCLE  Command failed with exit code 1.
```


### Payment link for reward

https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD&donation=true

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,10 @@
 # Dockerfile
 FROM node:18
 
-RUN npm i -g pnpm
+# Set the same pnpm version as the version in the package.json
+ENV PNPM_VERSION 8.6.0
+
+RUN npm i -g pnpm@$PNPM_VERSION
 WORKDIR /app
 
 COPY . .

